dagql/idtui/golden_test.go

```diff
@@ -106,6 +107,28 @@ func (s TelemetrySuite) TestGolden(ctx context.Context, t *testctx.T) {
 			"with-exec", "--args", "echo,hey",
 			"stdout",
 		}, Fail: true},
+
+		// tests intended to trigger consistent tui exec metrics output
+		{Function: "disk-metrics", Verbosity: 3, FuzzyTest: func(t *testctx.T, out string) {
```

Comment:
Even though `Verbosity` is set to `3` here, why are the results of the `disk-metrics` file not using it?

Reply:
that file should not have been committed, it is fully unused. see the comments on the example definition.